packages/relayer-lib/src/tx_builder/eth_to_cosmos.rs

```diff
@@ -353,12 +471,13 @@ where
             &self.eth_client,
             &ethereum_client_state.ibc_contract_address.to_string(),
             ethereum_client_state.ibc_commitment_slot,
-            target_block_number,
-            trusted_slot,
+            proof_block_number,
+            latest_trusted_slot,
         )
         .await?;
 
         let update_msgs = headers
+            .clone()
```

Comment:
You can remove this clone if you remove the len of the headers from the summary info. I think headers len is not super useful in info. It might be useful to know how many finality updates and period updates are included.

Reply:
Makes sense, will count those